### PR TITLE
APMrover2: remove unreachable code

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

File: APMrover2/test.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ int8_t Rover::test_passthru(uint8_t argc, const Menu::arg *argv)
             return (0);
         }
     }
-    return 0;
+
```
Comment (guludo) [inline APMrover2/test.cpp]: I'd rather replace line 95 with a break statement.

**Comment by khancyr** [comment]
@guludo, I like you solution.
Should I transform all test to use break instead of direct return ?

**Comment by guludo** [comment]
> Should I transform all test to use break instead of direct return ?

Oh, now I see that that's the way that is done in the file. Well, I *personally* like using `break` statement better. But I'm okay with using the current approach, since it's consistent with the whole file.

**Comment by khancyr** [comment]
Need to be rework :

> Bill Bonney @billbonney 16:31
> @khancyr You should also consider the fact if you modify a function, you need a way to verify your changes are good for potential code paths. The removal of the return is dead code, what warning or error is it fixing? some compilers in the past would have warning if that return wasn’t their (most modern ones are smarter for sure) The line of code you highlight was written in 2012.
> The actual issue seems that a command is reported as verify_command as true, when previously is was false https://github.com/ArduPilot/ardupilot/commit/4ca3a926559ac46bb9e4f289f625d94bf9ff368a#diff-67f5454129c542b1a46075d73ad59dd9L179
> Maybe the issue is that the behaviour was redefined, was that intential and the extra return true; for the default in the swicth is the line that should be removed?

**Comment by khancyr** [comment]
closed in favor of https://github.com/ArduPilot/ardupilot/pull/4711

**Pull request closed without merging**